lib/zendesk_apps_tools/common.rb

```diff
@@ -1,5 +1,36 @@
+# frozen_string_literal: true
 module ZendeskAppsTools
   module Common
+    module ClassMethods
+      def shared_options(except: [])
+        unless except.include? :path
+          method_option :path, {
+            type: :string,
+            default: './',
+            aliases: ['-p']
+          }
+        end
+        unless except.include? :clean
+          method_option :clean, {
+            type: :boolean,
+            default: false
+          }
+        end
+        unless except.include? :unattended
+          method_option :unattended, {
+            type: :boolean,
+            default: false,
+            desc: 'Never prompt for input, expecting all input from the original invocation. Many commands invoked '\
+                  'with this option will just crash.'
```

Comment:
> Many commands invoked with this option will just crash.

Is this because the command expects input that was not passed by the original invocation? I don't find that statement very helpful 😅